modules/indigo/module/inc/indigo/of_connection_manager.h

```diff
@@ -291,17 +290,20 @@ extern indigo_error_t indigo_cxn_connection_status_get(
 
 /**
  * Connection status change handler callback prototype
- * @param cxn_id The descriptor of the connection that has changed
- * @param state The new state of the connection
- * @param cookie Passed in on register
+ * @param controller_id The descriptor of the controller that has changed
+ * @param aux_id The connection's auxiliary identifier
+ * @param cxn_proto_params Protocol parameters associated with the controller
+ * @param is_connected True if the connection has completed handshake
+ * @param cookie Passed in to register function
  *
  * For complete information, do a status_get on the connection
  */
 
 typedef void (*indigo_cxn_status_change_f)(
```

Comment:
it's still a connection status change callback.  before, you had a cxn_id, but there wasn't much you could do with that cxn_id, since controller_add returns a controller_id.  now you at least get a controller_id, an aux id, and a pointer to proto params to let you figure out which connection has actually changed state.